### ISSUE TITLE
NSM Applications exported as LV2 plugin don't save there state.

### ISSUE THREAD

**Issue by brummer10** [falkTX/Carla#1263]
It would be nice if you could forward LV2 State save as NSM save to such exported apps to allow them to save there state in a host session.

**Comment by falkTX** [comment]
This is not related to bypassing NSM signal or not, but that the exported lv2 plugins (which I should remind is marked as experimental, because it is imcomplete) is missing support for lv2 state extension.

**Comment by brummer10** [comment]
Yes, I know it's experimental. I don't say or think anything is bypassed. This is a feature request. 

> forward LV2 State save as NSM save 

I just mention NSM save because I've no idea how you would forward LV2 State save to apps which don't support NSM.

**Comment by falkTX** [comment]
sure, I will do it eventually. the lv2 export was initially done as a test to see if it was possible at all, and how well it would behave.
seems like it works well, apart from missing state, so that is the next step now
